partner_compassion/views/partner_compassion_view.xml

```diff
@@ -92,6 +92,11 @@
                 <field name="user_login" attrs="{'invisible': [('user_login', '=', False)]}"/>
                 <field name="birthdate_date" attrs="{'invisible': [('is_company','=',True)]}"/>
                 <field name="deathdate" attrs="{'invisible': [('is_company','=',True)]}"/>
+
+                <field name="is_of_age" invisible="1"/>
+                <field name="age"/>
+                <field name="parent_consent" attrs="{'invisible': [('is_of_age','=',True)]}"/>
+                <field name="can_manage_paid_sponsorships" attrs="{'invisible': [('is_of_age','=',True)]}"/>
```

Comment:
I think this field can only stay internal to the code logic and doesn't need to be seen in the view.
```suggestion
```